lib/AI-Agent.xml

```diff
@@ -19,18 +19,18 @@
             <Method name="encryptFile"/>
         </Class>
         <Class name="uk.gov.hmcts.reform.sendletter.services.ftp.FtpClient">
-            <Method name="upload"/>
+            <Method name="upload" reportExecutionTime="true" signature="(Lnet/schmizz/sshj/xfer/LocalSourceFile;Z)V"/>
             <Method name="downloadReports"/>
-            <Method name="deleteReport"/>
+            <Method name="deleteReport" reportExecutionTime="true" signature="(Ljava/lang/String;)V"/>
```

Comment:
what's the actual fix?
adding `reportExecutionTime="true"` or specifying signature?
isn't signature only needed when there are two methods with the same name?